test/manipulation.test.js

```diff
@@ -688,6 +688,311 @@ describe('manipulation', function () {
     });
   });
 
+  if (!getSchema().connector.replaceOrCreate) {
```

Comment:
Based on our discussion, I think this should be `!getSchema().connector.replaceById`.